src/clouds.cpp

```diff
@@ -191,7 +193,10 @@ void Clouds::render()
 					(float)p_in_noise_i.X * cloud_size_noise,
 					(float)p_in_noise_i.Y * cloud_size_noise,
 					m_seed, 3, 0.5);
-			grid[i] = (noise >= 0.4);
+			// normalize to 0..1 (given 3 octaves)
+			double noise_bound = 1.0 + 0.5 + 0.25;
+			double density = noise / noise_bound * 0.5 + 0.5;
+			grid[i] = (density < m_density);
```

Comment:
So, previous code had 3 octaves 0.5 persistence and a threshold of cloud when noise > 0.4.
From my mapgen experience this results in about one third area, but i'll try to get an exact new threshold.